<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Manager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">BACKGROUND-COLOR: rgb(255,255,255);</string>
  </property>
  <widget class="QWidget" name="UF_UserManageridget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QLabel" name="LAB_Title">
      <property name="font">
       <font>
        <pointsize>26</pointsize>
        <underline>false</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">BACKGROUND-COLOR: rgb(208, 248, 255);
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;User Manager&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTableWidget" name="TAB_Users">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="PB_Back">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change your password, create new users, and delete existing users&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">BACKGROUND-COLOR: rgb(160, 238, 252);
BORDER-COLOR: rgb(160, 238. 252);
BORDER-RADIUS: 10px;
BORDER-STYLE: outset;
</string>
      </property>
      <property name="text">
       <string>Return to Login</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="PB_DeleteUser">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirm username and password to log in&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">BACKGROUND-COLOR: rgb(255, 175, 177);
BORDER-COLOR: rgb(255, 175. 177);
BORDER-RADIUS: 10px;
BORDER-STYLE: outset;</string>
      </property>
      <property name="text">
       <string>Delete User</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="PB_EditUser">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirm username and password to log in&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">BACKGROUND-COLOR: rgb(255, 238, 0);
BORDER-COLOR: rgb(170, 255, 127);
BORDER-RADIUS: 10px;
BORDER-STYLE: outset;
</string>
      </property>
      <property name="text">
       <string>Edit User</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="PB_AddUser">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Confirm username and password to log in&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">BACKGROUND-COLOR: rgb(170, 255, 127);
BORDER-COLOR: rgb(170, 255, 127);
BORDER-RADIUS: 10px;
BORDER-STYLE: outset;
</string>
      </property>
      <property name="text">
       <string>Add User</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>TAB_Users</tabstop>
  <tabstop>PB_AddUser</tabstop>
  <tabstop>PB_EditUser</tabstop>
  <tabstop>PB_DeleteUser</tabstop>
  <tabstop>PB_Back</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
